### DISCUSSION TITLE
QueryString forces the queryModification to use URL encoding

### DISCUSSION THREAD

**Discussion by ColonelSlanders** [microsoft-search/pnp-modern-search#3782] [Q&A]
I'm not sure if this is an issue or if this is something I can fix, please help me know.

I am using {QueryString.q} in my QueryTemplate in place of {searchResults} since users will need to copy/paste the URL for the search to share with other users. 

However, when I check the debug to see data > queryModification, I'm seeing my search text not encoded:

This is the result when I type: test case
into the search bar.
 "queryModification": "test%20case"

These are the settings for my Search Box, sending Queries to the page:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/171854481/ab194268-c8aa-46f3-b90d-ed724a31a360)

So when it searches for "test%2520case". it's not going to find anything with the URL encoded. How can I get this to be not encoded?

**Comment by wobba** [comment]
See https://microsoft-search.github.io/pnp-modern-search/usage/search-box/#connections where you consume the &q parameter in data binding.

**Reply by ColonelSlanders** [reply]
Thanks. Even after updating that to the Query String parameter in the Search Box, I still have the result.

I have found an issue that mentions 4.3.0 (our current enterprise version) that this feature does not work correctly. I am attempting to get IT to update this to a newer version.

Issue I'm referring to: https://github.com/microsoft-search/pnp-modern-search/issues/1236
